survey_resource_booking/tests/test_invite.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from datetime import datetime
+
+from freezegun import freeze_time
+
+from odoo.exceptions import UserError
+from odoo.tests.common import Form, TransactionCase
+
+from ...resource_booking.tests.common import create_test_data
```

Comment:
why don't U use instead:
```suggestion
from odoo.addons.resource_booking.tests.common import create_test_data
```